client/landing/stepper/index.tsx

```diff
@@ -136,6 +138,7 @@ window.AppBoot = async () => {
 	const reduxStore = createReduxStore( initialState, initialReducer );
 	setStore( reduxStore, getStateFromCache( userId ) );
 	setupLocale( user, reduxStore );
+	window.redux = reduxStore;
```

Comment:
It would be great to get rid of this. Would it be hard?

Reply:
I used it for https://github.com/Automattic/wp-calypso/blob/update/tailored-flows-domains-step-backup/client/lib/signup/step-actions/fetch-sites-and-user.js#L11, but we're not currently using in the flow and it's working.
Maybe it was used to have a faster start for the user after the purchase? I think we can get rid of it here.

Reply:
Yup nuke it.